gateway-ha/src/main/java/io/trino/gateway/ha/router/HaGatewayManager.java

```diff
@@ -82,13 +83,25 @@ public Optional<ProxyBackendConfiguration> getBackendByName(String name)
     @Override
     public void deactivateBackend(String backendName)
     {
-        dao.deactivate(backendName);
+        GatewayBackend model = dao.findFirstByName(backendName);
+        if (model != null) {
+            Boolean prevStatus = model.active();
+            dao.deactivate(backendName);
+            log.info("Backend cluster %s has been deactivated (previous status: active=%s).",
+                    backendName, prevStatus);
```

Comment:
For `updateBackend` we only log if there was an actual change, but here we log regardless. Let's be consistent across all methods, either always logging or only logging if there is a change in state.